requirements.txt

```diff
@@ -6,4 +6,4 @@ dill>=0.3
 python-dateutil>=2.8.0
 stevedore>=3.0.0
 typing-extensions
-symengine>=0.11
```

Comment:
It might be better to revert this change so that the people looking for where `symengine>=0.11` was introduced don't come back to this PR 🙂